lib/benchee/conversion/count.ex

```diff
@@ -111,16 +113,24 @@ defmodule Benchee.Conversion.Count do
   ## Examples
 
       iex> Benchee.Conversion.Count.best([23, 23_000, 34_000, 2_340_000])
-      :thousand
+      %Benchee.Conversion.Unit{long:      "Thousand",
+                               magnitude: 1000,
+                               short:     "K"}
```

Comment:
Thinking about the fields of the`Unit` struct, now that it's "exposed" (I see that it's not _actually_ exposed in the interface, but it's all over the doctests now.

It would be nice for the doctests to be really clean and simple. Some ideas:

- rename "short" to "label"
```
iex> Benchee.Conversion.Count.best([23, 23_000, 34_000, 2_340_000]).label
"K"
```
- add a "name" field: `%Unit{name: :thousand}`
```
iex> Benchee.Conversion.Count.best([23, 23_000, 34_000, 2_340_000]).name
:thousand
```

It might make sense to omit `:long` since it is never used.